File: src/Generated20260226.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260226 {
public:
    // Generated class 20260226_194401_188832
    int value = 353;
    string created = "20260226_194401_188832";
    
    int process() {
        int result = value;
        for (int i = 0; i < 38; i++) {
            result += i * 8;
        }
        return result;
    }
};

int main() {
    Generated20260226 obj;
    cout << obj.process() << endl;
    return 0;
}

File: src/Generated20260226.go
package main

import "fmt"

type Generated20260226 struct {
    // Generated struct 20260226_194543_767495
    Value int
    Created string
}

func (c *Generated20260226) Process() int {
    result := c.Value
    for i := 0; i < 31; i++ {
        result += i * 6
    }
    return result
}

func main() {
    obj := Generated20260226{Value: 255, Created: "20260226_194543_767495"}
    fmt.Println(obj.Process())
}